src/nvim/memory.c

```diff
@@ -333,43 +333,16 @@ size_t memcnt(const void *data, char c, size_t len)
   return cnt;
 }
 
-/// Copies the string pointed to by src (including the terminating NUL
-/// character) into the array pointed to by dst.
-///
-/// @returns pointer to the terminating NUL char copied into the dst buffer.
-///          This is the only difference with strcpy(), which returns dst.
-///
-/// WARNING: If copying takes place between objects that overlap, the behavior
-/// is undefined.
-///
-/// Nvim version of POSIX 2008 stpcpy(3). We do not require POSIX 2008, so
-/// implement our own version.
-///
-/// @param dst
-/// @param src
+#ifndef HAVE_STPCPY
 char *xstpcpy(char *restrict dst, const char *restrict src)
```

Comment:
why delete the entire doc? at least a note like this is helpful:

    /// Nvim version of POSIX 2008 stpcpy(3).

Reply:
I was following xstrnlen which has no comments, presumably because it follows POSIX strnlen.

Reply:
xstrnlen could have a similiar docstring. A 1-line comment providing a reference is low-cost way to give hints.